powertools-e2e-tests/handlers/idempotency/pom.xml

```diff
@@ -0,0 +1,98 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>software.amazon.lambda</groupId>
+        <artifactId>e2e-test-handlers-parent</artifactId>
+        <version>1.0.0</version>
+    </parent>
+
+    <artifactId>e2e-test-handler-idempotency</artifactId>
+    <packaging>jar</packaging>
+    <name>A Lambda function using powertools idempotency</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>software.amazon.lambda</groupId>
+            <artifactId>powertools-idempotency</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
```

Comment:
Do we need this, or is it taken transitively through powertools-idempotency?

Reply:
I think lambda-java-core is, you are right. Same for the others.